js/package.json

```diff
@@ -101,12 +100,13 @@
     "source-map-loader": "0.2.4",
     "terser-webpack-plugin": "1.2.1",
     "trash": "4.3.0",
-    "ts-jest": "23.10.5",
-    "ts-node": "8.0.2",
+    "ts-jest": "24.0.2",
+    "ts-node": "8.2.0",
     "tslint": "5.12.1",
```

Comment:
At some pint, we should replace tslint with slint since the former is being deprecated.

Reply:
yes I imagine at some point we should switch over.